NEWS.md

```diff
@@ -4,11 +4,12 @@ TrixiParticles.jl follows the interpretation of [semantic versioning (semver)](h
 used in the Julia ecosystem. Notable changes will be documented in this file for human readability.
 We aim at 3 to 4 month between major release versions and about 2 weeks between minor versions.
 
-## Version 0.3.x

```

Comment:
## Version 0.3.x

### Highlights

### Added
Particle sampling for complex geometries from `.stl` and `.asc` files.

### Removed

### Deprecated

## Version 0.2.1

### Highlights
Particle sampling for geometries from `.stl` and `.asc` files.